tv_apps/smart-system/js/sync_manager.js

```diff
@@ -402,6 +402,11 @@
       // to do it for a periodic sync.
       this.unregisterSyncRequest();
 
+      if (!navigator.onLine) {
```

Comment:
Should we trigger an offline error here? Or leaving it to the ```Enabled``` state is intended?